### DISCUSSION TITLE
Dim unused parameters?

### DISCUSSION THREAD

**Discussion by fgnAT** [ykrx/Lightbulb#8] [Ideas]
Add a feature to visually "dim" unused function parameters in the editor, making them appear less prominent (e.g., with a gray color or italic style).

**Comment by ykrx** [comment]
Hey, new release supports this now :) via `editorUnnecessaryCode.opacity`

<img width="908" alt="image" src="https://github.com/user-attachments/assets/74689035-a660-46cc-b0af-ed92dd3b9105" />
